druzhba/postgres.py

```diff
@@ -281,6 +278,20 @@ def query(self, sql):
                 for dict_row in cursor:
                     yield dict_row
 
+    def _get_column_size(self, column_type, internal_size, precision, scale):
+        if self.type_map.get(column_type, column_type).lower() == "decimal":
```

Comment:
I actually checked `pg_type` on a local PostgreSQL and only saw `numeric` in there, no `decimal`. It might be safer to just do `in ('decimal', 'numeric')`?

Reply:
Good point.